exposed.go

```diff
@@ -0,0 +1,27 @@
+package exposed
+
+import (
+	"errors"
+	//auto register default codecs
+)
+
+var (
+	//ErrTransport at transport level
+	ErrTransport = errors.New("transport error")
+	//error at operation handler layer
+	ErrOperation = errors.New("operation error")
+	//Err operation already registred
```

Comment:
comment on exported var ErrOperationAlreadyRegistered should be of the form "ErrOperationAlreadyRegistered ..."